tests/embind/build_benchmark

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-../../emcc --minify 0 --bind --post-js embind.benchmark.js -O2 --shell-file shell.html -o embind_benchmark.html embind_benchmark.cpp
+../../emcc -sEXPORTED_FUNCTIONS=_main,_get_counter,_increment_counter,_returns_input,_sum_int,_sum_float --minify 0 --bind --js-library=embind.benchmark.js -O2 --shell-file shell.html -o embind_benchmark.html embind_benchmark.cpp
```

Comment:
Was converting this file from a post-js to a js-library necessary? 

Perhaps its was the case that providing externs to C/C++ was possible via a post-js in previous version of emscripten?  I wonder if `-sWARN_ON_UNDEFINED_FUNCTIONS=0` would have allowed the post-js method to continue to work?  Otherwise, how did it ever work?

Reply:
I was wondering how this use to work. Is one way or the other preferred?

Reply:
`-sWARN_ON_UNDEFINED_SYMBOLS=0` works, do you want me to revert these changes?

Reply:
No, its better to use `--js-library` file I think.